routes/users.js

```diff
@@ -12,9 +12,9 @@ const {
     resetWithToken,
     validateEmail,
 } = userControl;
-
+const { UserValidation } = userValidate;
 // Register Routes
-router.post('/register', registerUser);
+router.post('/register', UserValidation, registerUser);
```

Comment:
This line crashes the application.
![application crash](https://user-images.githubusercontent.com/38657595/60873724-7de04580-a23f-11e9-8ba6-562fee34e9ec.jpg)